python/grass/jupyter/interact_display.py

```diff
@@ -70,70 +67,75 @@ def __init__(self, width=400, height=400):
         # Create LayerControl default
         self.layer_control = False
 
-    def _convert_coordinates(self, x, y, proj_in):
-        """This function reprojects coordinates to WGS84, the required
-        projection for vectors in folium.
-
-        Arguments:
-            x -- x coordinate (string)
-            y -- y coordinate (string)
-            proj_in -- proj4 string of location (for example, the output
-            of g.region run with the `g` flag."""
-
-        # Reformat input
-        coordinates = f"{x}, {y}"
-        # Reproject coordinates
-        coords_folium = gs.read_command(
-            "m.proj",
-            coordinates=coordinates,
-            proj_in=proj_in,
-            separator="comma",
-            flags="do",
+    def _setup_vector_location(self):
+        # Create new vector environment for tmp WGS84 location
+        rcfile, self._wgs84_env = gs.create_environment(
```

Comment:
I think you are right but I have all the cleanup in PR #1754. I think it makes most sense to move rcfile cleanup here and delete that PR since it (1) doesn't contain much new code and (2) this PR contains so many big changes, it's just going to cause a bunch of merge conflicts.